static/app/views/detectors/components/detectorDetailsSidebar.tsx

```diff
@@ -0,0 +1,246 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {GroupPriorityBadge} from 'sentry/components/badge/groupPriority';
+import {Flex} from 'sentry/components/core/layout';
+import {Tooltip} from 'sentry/components/core/tooltip';
+import {DateTime} from 'sentry/components/dateTime';
+import {KeyValueTable, KeyValueTableRow} from 'sentry/components/keyValueTable';
+import Placeholder from 'sentry/components/placeholder';
+import TextOverflow from 'sentry/components/textOverflow';
+import TimeSince from 'sentry/components/timeSince';
+import Section from 'sentry/components/workflowEngine/ui/section';
+import {IconArrow} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {PriorityLevel} from 'sentry/types/group';
+import {
+  DataConditionType,
+  DetectorPriorityLevel,
+} from 'sentry/types/workflowEngine/dataConditions';
+import type {Detector} from 'sentry/types/workflowEngine/detectors';
+import {useTeamsById} from 'sentry/utils/useTeamsById';
+import useUserFromId from 'sentry/utils/useUserFromId';
+import DetailsPanel from 'sentry/views/detectors/components/detailsPanel';
+import {getResolutionDescription} from 'sentry/views/detectors/utils/getDetectorResolutionDescription';
+
+const DETECTOR_PRIORITY_LEVEL_TO_PRIORITY_LEVEL: Record<
+  DetectorPriorityLevel,
+  PriorityLevel
+> = {
+  [DetectorPriorityLevel.LOW]: PriorityLevel.LOW,
+  [DetectorPriorityLevel.MEDIUM]: PriorityLevel.MEDIUM,
+  [DetectorPriorityLevel.HIGH]: PriorityLevel.HIGH,
+  [DetectorPriorityLevel.OK]: PriorityLevel.LOW,
+};
+
+function getDetectorEnvironment(detector: Detector) {
+  return (
+    detector.dataSources?.find(ds => ds.type === 'snuba_query_subscription')?.queryObj
+      ?.snubaQuery.environment ?? t('All environments')
+  );
+}
+
+function AssignToTeam({teamId}: {teamId: string}) {
+  const {teams} = useTeamsById({ids: [teamId]});
```

Comment:
Should we show a placeholder here for the team like we are for the user? Pretty sure we only load the first 100 teams or something when the app loads